hpx/compute/cuda/default_executor.hpp

```diff
@@ -188,6 +188,12 @@ namespace hpx { namespace compute { namespace cuda
             target_.synchronize();
         }
 
+        template <typename F, typename ... Ts>
+        void execute(F && f, Ts &&... ts) const
+        {
+            sync_execute(std::forward<F>(f), std::forward<Ts>(ts)...);
+        }
+
```

Comment:
Ok, let's rather change that spot to use `sync_execute()`. I must have missed changing that while re-implementing the executors.